src/test/java/com/wikia/webdriver/testcases/mobile/CommentsTests.java

```diff
@@ -35,106 +39,55 @@
 
   private CommentsPageObject comments;
 
-  private void init() {
-    this.comments = new CommentsPageObject(driver);
-
-    new Navigate().toPage(MobileSubpages.COMMENTS);
-  }
-
   @Test(groups = "mercury_comments_containsAvatarUsernameTimestampAndContent")
   public void mercury_comments_containsAvatarUsernameTimestampAndContent() {
-    init();
+    this.comments = new CommentsPageObject(driver);
+    new Navigate().toPage(MobileSubpages.COMMENTS);
 
-    Assertion.assertTrue(
-        comments.isCommentsListCollapsed(),
-        "Comments are expanded"
-    );
+    Assertion.assertTrue(comments.isCommentsListCollapsed(), "Comments are expanded");
 
-    Log.log(
-        "Comments list",
-        "is collapsed",
-        true
-    );
+    Log.log("Comments list", "is collapsed", true);
 
     comments.clickCommentsHeader();
     comments.waitForCommentsToLoad();
 
-    Assertion.assertFalse(
-        comments.isCommentsListCollapsed(),
-        "Comments are collapsed"
-    );
+    Assertion.assertFalse(comments.isCommentsListCollapsed(), "Comments are collapsed");
 
-    Log.log(
-        "Comments list",
-        "is expanded",
-        true
-    );
+    Log.log("Comments list", "is expanded", true);
 
     boolean result = comments.getNumberOfCommentsPerPage() == NUMBER_OF_COMMENTS_PER_PAGE;
-    Log.log(
-        "Number of comments per page",
-        "is correct",
-        "is incorrect",
-        result
-    );
+    Log.log("Number of comments per page", "is correct", "is incorrect",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "is correct" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)